react/features/conference/components/native/NavigationBar.js

```diff
@@ -45,45 +52,43 @@ type Props = {
  * @param {Props} props - The React props passed to this component.
  * @returns {React.Node}
  */
-const NavigationBar = (props: Props) => {
-    if (!props._visible) {
-        return null;
-    }
-
-    return (
+const NavigationBar = (props: Props) => (<>
+    {props._visible && <View
+        pointerEvents = 'box-none'
+        style = { styles.navBarWrapper }>
+        <View style = { styles.pipButtonContainer }>
+            <PictureInPictureButton styles = { styles.pipButton } />
+        </View>
         <View
             pointerEvents = 'box-none'
-            style = { styles.navBarWrapper }>
-            <View style = { styles.pipButtonContainer }>
-                <PictureInPictureButton styles = { styles.pipButton } />
-            </View>
-            <View
-                pointerEvents = 'box-none'
-                style = { styles.roomNameWrapper }>
-                {
-                    props._meetingNameEnabled
+            style = { styles.roomNameWrapper }>
+            {
+                props._conferenceTimerEnabled
+                    && <View style = { styles.roomTimerView }>
+                        <ConferenceTimer textStyle = { styles.roomTimer } />
+                    </View>
+            }
+            {
+                props._meetingNameEnabled
                         && <View style = { styles.roomNameView }>
                             <Text
                                 numberOfLines = { 1 }
                                 style = { styles.roomName }>
                                 { props._meetingName }
                             </Text>
                         </View>
-                }
-                {
-                    props._conferenceTimerEnabled
-                            && <View style = { styles.roomTimerView }>
-                                <ConferenceTimer textStyle = { styles.roomTimer } />
-                            </View>
-                }
-                <Labels />
-            </View>
-            <View style = { styles.inviteButtonContainer }>
-                <InviteButton styles = { styles.inviteButton } />
-            </View>
+            }
+            {/* eslint-disable-next-line react/jsx-no-bind */}
+            <Labels createOnPress = { props._createOnPress } />
         </View>
-    );
-};
+        <View style = { styles.titleBarButtonContainer }>
```

Comment:
I think it's better if from NavigationBar we change to TitleBar. 
Also, style definitions should be updated to that.